### PR TITLE
Roadmap item: platform categories, take 2

### PR DESCRIPTION
**Pull request by levitte**
Our roadmap says this:

> Platform Strategy
> 1.  Classify all the platforms known by Configure according to the  [platform policy](https://www.openssl.org/policies/platformpolicy.html) (Timescale: Next feature release)
> 2.  Publish the build and test status for each platform (Timescale: Next feature release)

This PR attempts to address item 1, and is an alternative to #5043

### CODE REVIEW

**Comment by levitte** [comment]
I made this variant because #5043 caused a fragmentation into diverse files that I found confusing.
I'd rather support this form.

**Comment by levitte** [comment]
Please have a look through the classifications I made.  I *think* I got everything from #5043 as well as private or OMC emails.

File: Configurations/README [hunk 1]
```diff
@@ -35,6 +39,54 @@ In each table entry, the following keys are significant:
...
+                           "community"  This is a config target
+                                        supported by at least one
+                                        named (self defined) member of
+                                        the OpenSSL community.
+                           "unknown"    This is a config target that
+                                        does not have a named
+                                        supporter.
+                           "deprecated" This is a config target that
+                                        has been deemed legacy and is
+                                        subject for future
```
Comment (richsalz) [inline Configurations/README]: maybe "and could be removed in the future" is more clear?

File: Configurations/README [hunk 2]
```diff
@@ -35,6 +39,54 @@ In each table entry, the following keys are significant:
...
+
+                           Note template configs are usually
+                           unclassified, as well as some very generic
+                           ones.
+
+        supporters      => A list of email addresses, perferably but
+                           not necessarily with a real name.  The
+                           presence of this attribute is required for
+                           config targets classified as "secondary" or
+                           "community".
```
Comment (richsalz) [inline Configurations/README]: And what happens with config if the field isn't there?

**Review by richsalz — COMMENTED** [review]
Interesting idea; a comment and a question.

File: Configurations/README (hunk 1 above)
Comment (levitte) [reply Configurations/README]: I actually meant to write "for future removal".  Would that be suitable enough?

Comment (richsalz) [reply Configurations/README]: certainly!

File: Configurations/README (hunk 2 above)
Comment (levitte) [reply Configurations/README]: These attributes are informative, so builds will continue to work as they have worked so far.

However, `./Configure SUPPORT` will write a line like this for a secondary or community target with no named supporters:

```
foo                           secondary       [[ NO SUPPORTERS ]]
bar                           community       [[ NO SUPPORTERS ]]
```

**Comment by levitte** [comment]
`./Configure SUPPORT` currently says this on my machine (with a couple of fake config targets "foo" and "bar"):

```
: ; ./Configure SUPPORT 
Using implicit seed configuration
BS2000-OSD                    unknown
BSD-generic32                 secondary       Ben Kaduk <kaduk@mit.edu>
BSD-generic64                 secondary       Ben Kaduk <kaduk@mit.edu>
BSD-ia64                      unknown
BSD-sparc64                   unknown
BSD-sparcv8                   unknown
BSD-x86                       secondary       Ben Kaduk <kaduk@mit.edu>
BSD-x86-elf                   secondary       Ben Kaduk <kaduk@mit.edu>
BSD-x86_64                    secondary       Ben Kaduk <kaduk@mit.edu>
Cygwin                        unknown
Cygwin-i386                   unknown
Cygwin-i486                   unknown
Cygwin-i586                   unknown
Cygwin-i686                   unknown
Cygwin-x86                    unknown
Cygwin-x86_64                 unknown
DJGPP                         unknown
MPE/iX-gcc                    unknown
QNX6                          unknown
QNX6-i386                     unknown
UEFI                          unknown
UWIN                          unknown
VC-CE                         unknown
VC-WIN32                      secondary       Richard Levitte <levitte@openssl.org>
                                              Paul Dale <paul.dale@oracle.com>
VC-WIN64A                     secondary       Richard Levitte <levitte@openssl.org>
                                              Paul Dale <paul.dale@oracle.com>
VC-WIN64A-masm                unknown
VC-WIN64I                     secondary       Paul Dale <paul.dale@oracle.com>
aix-cc                        secondary       appro@openssl.org
aix-gcc                       secondary       appro@openssl.org
aix64-cc                      secondary       appro@openssl.org
aix64-gcc                     secondary       appro@openssl.org
android                       secondary       appro@openssl.org
android-armeabi               secondary       appro@openssl.org
android-mips                  secondary       appro@openssl.org
android-x86                   secondary       appro@openssl.org
android64                     secondary       appro@openssl.org
android64-aarch64             secondary       appro@openssl.org
android64-mips64              secondary       appro@openssl.org
android64-x86_64              secondary       appro@openssl.org
bar                           community       [[ NO SUPPORTERS ]]
bsdi-elf-gcc                  unknown
cc                            generic
darwin-i386-cc                unknown
darwin-ppc-cc                 unknown
darwin64-debug-test-64-clang  primary         The OpenSSL Team
darwin64-ppc-cc               unknown
darwin64-x86_64-cc            secondary       Rich Salz <rsalz@openssl.org>
                                              Viktor Dukhovni <viktor@openssl.org>
debug                         generic
debug-erbridge                unknown
debug-linux-ia32-aes          primary         The OpenSSL Team
debug-linux-pentium           primary         The OpenSSL Team
debug-linux-ppro              primary         The OpenSSL Team
debug-test-64-clang           primary         The OpenSSL Team
dist                          generic
foo                           secondary       [[ NO SUPPORTERS ]]
gcc                           generic
haiku-x86                     unknown
haiku-x86_64                  unknown
hpux-ia64-cc                  secondary       appro@openssl.org
                                              Paul Dale <paul.dale@oracle.com>
hpux-ia64-gcc                 secondary       appro@openssl.org
                                              Paul Dale <paul.dale@oracle.com>
hpux-parisc-cc                community       appro@openssl.org
hpux-parisc-gcc               community       appro@openssl.org
hpux-parisc1_1-cc             community       appro@openssl.org
hpux-parisc1_1-gcc            community       appro@openssl.org
hpux64-ia64-cc                secondary       appro@openssl.org
                                              Paul Dale <paul.dale@oracle.com>
hpux64-ia64-gcc               secondary       appro@openssl.org
                                              Paul Dale <paul.dale@oracle.com>
hpux64-parisc2-cc             community       appro@openssl.org
hpux64-parisc2-gcc            community       appro@openssl.org
hurd-x86                      unknown
ios-cross                     unknown
ios64-cross                   unknown
iphoneos-cross                unknown
irix-mips3-cc                 deprecated
irix-mips3-gcc                deprecated
irix64-mips4-cc               deprecated
irix64-mips4-gcc              deprecated
linux-aarch64                 primary         The OpenSSL Team
linux-alpha-gcc               primary         The OpenSSL Team
linux-aout                    primary         The OpenSSL Team
linux-arm64ilp32              primary         The OpenSSL Team
linux-armv4                   primary         The OpenSSL Team
linux-c64xplus                primary         The OpenSSL Team
linux-elf                     primary         The OpenSSL Team
linux-generic32               primary         The OpenSSL Team
linux-generic64               primary         The OpenSSL Team
linux-ia64                    primary         The OpenSSL Team
linux-mips32                  primary         The OpenSSL Team
linux-mips64                  primary         The OpenSSL Team
linux-ppc                     primary         The OpenSSL Team
linux-ppc64                   primary         The OpenSSL Team
linux-ppc64le                 secondary       Paul Dale <paul.dale@oracle.com>
linux-sparcv8                 secondary       Paul Dale <paul.dale@oracle.com>
linux-sparcv9                 secondary       Paul Dale <paul.dale@oracle.com>
linux-x32                     primary         The OpenSSL Team
linux-x86                     primary         The OpenSSL Team
linux-x86-clang               primary         The OpenSSL Team
linux-x86_64                  primary         The OpenSSL Team
linux-x86_64-clang            primary         The OpenSSL Team
linux32-s390x                 secondary       Paul Dale <paul.dale@oracle.com>
linux64-mips64                primary         The OpenSSL Team
linux64-s390x                 secondary       Paul Dale <paul.dale@oracle.com>
linux64-sparcv9               secondary       Paul Dale <paul.dale@oracle.com>
mingw                         secondary       appro@openssl.org
                                              Richard Levitte <levitte@openssl.org>
mingw64                       secondary       appro@openssl.org
                                              Richard Levitte <levitte@openssl.org>
nextstep                      unknown
nextstep3.3                   unknown
purify                        generic
qnx4                          unknown
sco5-cc                       deprecated
sco5-gcc                      deprecated
solaris-sparcv7-cc            unknown
solaris-sparcv7-gcc           unknown
solaris-sparcv8-cc            secondary       Paul Dale <paul.dale@oracle.com>
solaris-sparcv8-gcc           secondary       Paul Dale <paul.dale@oracle.com>
solaris-sparcv9-cc            secondary       Paul Dale <paul.dale@oracle.com>
solaris-sparcv9-gcc           secondary       Paul Dale <paul.dale@oracle.com>
solaris-x86-gcc               secondary       Paul Dale <paul.dale@oracle.com>
solaris64-sparcv9-cc          secondary       Paul Dale <paul.dale@oracle.com>
solaris64-sparcv9-gcc         secondary       Paul Dale <paul.dale@oracle.com>
solaris64-x86_64-cc           secondary       Paul Dale <paul.dale@oracle.com>
solaris64-x86_64-gcc          secondary       Paul Dale <paul.dale@oracle.com>
tru64-alpha-cc                deprecated
tru64-alpha-gcc               deprecated
uClinux-dist                  unknown
uClinux-dist64                unknown
unixware-2.0                  deprecated
unixware-2.1                  deprecated
unixware-7                    deprecated
unixware-7-gcc                deprecated
vms-alpha                     secondary       Richard Levitte <levitte@openssl.org>
vms-alpha-p32                 secondary       Richard Levitte <levitte@openssl.org>
vms-alpha-p64                 secondary       Richard Levitte <levitte@openssl.org>
vms-ia64                      secondary       Richard Levitte <levitte@openssl.org>
vms-ia64-p32                  secondary       Richard Levitte <levitte@openssl.org>
vms-ia64-p64                  secondary       Richard Levitte <levitte@openssl.org>
vos-gcc                       unknown
vxworks-mips                  unknown
vxworks-ppc405                unknown
vxworks-ppc60x                unknown
vxworks-ppc750                unknown
vxworks-ppc750-debug          unknown
vxworks-ppc860                unknown
vxworks-ppcgen                unknown
vxworks-simlinux              unknown
```

**Comment by t-j-h** [comment]
Not all Linux targets are primary. We only noted x86 varients when we discussed that.
Can you point to who indicated they are using the others for daily development?

**Comment by levitte** [comment]
> Not all Linux targets are primary. We only noted x86 varients when we discussed that.

The policy says this, verbatim:

>> Primary
>> Target(s) on which the majority of OpenSSL development occurs
>> *The current primary development platform is Linux.*

So I'd say the policy needs to be redacted.

> Can you point to who indicated they are using the others for daily development?

I cannot.  However, I can point out the comment `# speculative`, which I placed on the config targets I was unsure of, since the platform policy that we have on display didn't add up with what I remember.

Are you saying that all those I've marked with `# speculative` should be changed to "unknown" until further notice?  Can we make sure to redact our written platform policy accordingly?

**Comment by t-j-h** [comment]
The platform policy needs to be updated to also note the specific target list - that is what we discussed - rather than the ultra high-level way we expressed it back in 2014. The definition was clear with "Target(s) on which the majority of OpenSSL development occurs" ... 

We had many discussions around this and many proposals as to how to handle - but the policy remains high-level. I think the definitions actually do make sense - but the specifics in the platform policy in terms of which platforms needs to be written out more clearly.

i.e. I would see it stated as Linux (specific target list here) ... and you will probably also end up with Linux (specific other-target-list) in Secondary as well - and some probably in other classificiations (like the 'alpha' platforms unless someone is volunteering to handle issues on those). 

I like the progress on this ... 

But don't forget that *deprecated* means we *plan* to remove. I would tend to put those into *unknown* at this stage ...

**Comment by levitte** [comment]
There are already some Linux that I've put down as secondary, specifically on request.

But ok, I'll move all those marked with `# speculative` to "unknown", I think that will satisfy everyone 'til further notice.

Regarding "deprecated", I've only marked those where someone (mostly @dot-asm) has said so.  If you have a different opinion, here's as good a place as any to voice it.  I would prefer a config target by config target comment if you have a differing view.  That'll keep us from having to guess exactly what we mean.

**Comment by levitte** [comment]
I've taken this out of WIP, I think it's time it gets its final review.

**Comment by levitte** [comment]
Currently on my machine, with this branch rebased on the latest master

```
: ; ./Configure SUPPORT
BS2000-OSD                    unknown
BSD-generic32                 secondary       Ben Kaduk <kaduk@mit.edu>
BSD-generic64                 secondary       Ben Kaduk <kaduk@mit.edu>
BSD-ia64                      unknown
BSD-sparc64                   unknown
BSD-sparcv8                   unknown
BSD-x86                       secondary       Ben Kaduk <kaduk@mit.edu>
BSD-x86-elf                   secondary       Ben Kaduk <kaduk@mit.edu>
BSD-x86_64                    secondary       Ben Kaduk <kaduk@mit.edu>
Cygwin                        unknown
Cygwin-i386                   unknown
Cygwin-i486                   unknown
Cygwin-i586                   unknown
Cygwin-i686                   unknown
Cygwin-x86                    unknown
Cygwin-x86_64                 unknown
DJGPP                         unknown
MPE/iX-gcc                    unknown
QNX6                          unknown
QNX6-i386                     unknown
UEFI                          unknown
UWIN                          unknown
VC-CE                         unknown
VC-WIN32                      secondary       Richard Levitte <levitte@openssl.org>
                                              Paul Dale <paul.dale@oracle.com>
VC-WIN64A                     secondary       Richard Levitte <levitte@openssl.org>
                                              Paul Dale <paul.dale@oracle.com>
VC-WIN64A-masm                unknown
VC-WIN64I                     secondary       Paul Dale <paul.dale@oracle.com>
aix-cc                        secondary       appro@openssl.org
aix-gcc                       secondary       appro@openssl.org
aix64-cc                      secondary       appro@openssl.org
aix64-gcc                     secondary       appro@openssl.org
android                       secondary       appro@openssl.org
android-armeabi               secondary       appro@openssl.org
android-mips                  secondary       appro@openssl.org
android-x86                   secondary       appro@openssl.org
android64                     secondary       appro@openssl.org
android64-aarch64             secondary       appro@openssl.org
android64-mips64              secondary       appro@openssl.org
android64-x86_64              secondary       appro@openssl.org
bar                           community       [[ NO SUPPORTERS ]]
bsdi-elf-gcc                  unknown
cc                            generic
darwin-i386-cc                unknown
darwin-ppc-cc                 unknown
darwin64-debug-test-64-clang  primary         The OpenSSL Team
darwin64-ppc-cc               unknown
darwin64-x86_64-cc            secondary       Rich Salz <rsalz@openssl.org>
                                              Viktor Dukhovni <viktor@openssl.org>
debug                         generic
debug-erbridge                unknown
debug-linux-ia32-aes          primary         The OpenSSL Team
debug-linux-pentium           primary         The OpenSSL Team
debug-linux-ppro              primary         The OpenSSL Team
debug-test-64-clang           primary         The OpenSSL Team
dist                          generic
foo                           secondary       [[ NO SUPPORTERS ]]
gcc                           generic
haiku-x86                     unknown
haiku-x86_64                  unknown
hpux-ia64-cc                  secondary       appro@openssl.org
                                              Paul Dale <paul.dale@oracle.com>
hpux-ia64-gcc                 secondary       appro@openssl.org
                                              Paul Dale <paul.dale@oracle.com>
hpux-parisc-cc                community       appro@openssl.org
hpux-parisc-gcc               community       appro@openssl.org
hpux-parisc1_1-cc             community       appro@openssl.org
hpux-parisc1_1-gcc            community       appro@openssl.org
hpux64-ia64-cc                secondary       appro@openssl.org
                                              Paul Dale <paul.dale@oracle.com>
hpux64-ia64-gcc               secondary       appro@openssl.org
                                              Paul Dale <paul.dale@oracle.com>
hpux64-parisc2-cc             community       appro@openssl.org
hpux64-parisc2-gcc            community       appro@openssl.org
hurd-x86                      unknown
ios-cross                     unknown
ios64-cross                   unknown
iphoneos-cross                unknown
irix-mips3-cc                 deprecated
irix-mips3-gcc                deprecated
irix64-mips4-cc               deprecated
irix64-mips4-gcc              deprecated
linux-aarch64                 unknown
linux-alpha-gcc               unknown
linux-aout                    unknown
linux-arm64ilp32              unknown
linux-armv4                   unknown
linux-c64xplus                unknown
linux-elf                     unknown
linux-generic32               primary         The OpenSSL Team
linux-generic64               primary         The OpenSSL Team
linux-ia64                    unknown
linux-mips32                  unknown
linux-mips64                  unknown
linux-ppc                     unknown
linux-ppc64                   unknown
linux-ppc64le                 secondary       Paul Dale <paul.dale@oracle.com>
linux-sparcv8                 secondary       Paul Dale <paul.dale@oracle.com>
linux-sparcv9                 secondary       Paul Dale <paul.dale@oracle.com>
linux-x32                     unknown
linux-x86                     primary         The OpenSSL Team
linux-x86-clang               primary         The OpenSSL Team
linux-x86_64                  primary         The OpenSSL Team
linux-x86_64-clang            primary         The OpenSSL Team
linux32-s390x                 secondary       Paul Dale <paul.dale@oracle.com>
linux64-mips64                unknown
linux64-s390x                 secondary       Paul Dale <paul.dale@oracle.com>
linux64-sparcv9               secondary       Paul Dale <paul.dale@oracle.com>
mingw                         secondary       appro@openssl.org
                                              Richard Levitte <levitte@openssl.org>
mingw64                       secondary       appro@openssl.org
                                              Richard Levitte <levitte@openssl.org>
nextstep                      unknown
nextstep3.3                   unknown
purify                        generic
qnx4                          unknown
sco5-cc                       deprecated
sco5-gcc                      deprecated
solaris-sparcv7-cc            unknown
solaris-sparcv7-gcc           unknown
solaris-sparcv8-cc            secondary       Paul Dale <paul.dale@oracle.com>
solaris-sparcv8-gcc           secondary       Paul Dale <paul.dale@oracle.com>
solaris-sparcv9-cc            secondary       Paul Dale <paul.dale@oracle.com>
solaris-sparcv9-gcc           secondary       Paul Dale <paul.dale@oracle.com>
solaris-x86-gcc               secondary       Paul Dale <paul.dale@oracle.com>
solaris64-sparcv9-cc          secondary       Paul Dale <paul.dale@oracle.com>
solaris64-sparcv9-gcc         secondary       Paul Dale <paul.dale@oracle.com>
solaris64-x86_64-cc           secondary       Paul Dale <paul.dale@oracle.com>
solaris64-x86_64-gcc          secondary       Paul Dale <paul.dale@oracle.com>
tru64-alpha-cc                deprecated
tru64-alpha-gcc               deprecated
uClinux-dist                  unknown
uClinux-dist64                unknown
unixware-2.0                  deprecated
unixware-2.1                  deprecated
unixware-7                    deprecated
unixware-7-gcc                deprecated
vms-alpha                     secondary       Richard Levitte <levitte@openssl.org>
vms-alpha-p32                 secondary       Richard Levitte <levitte@openssl.org>
vms-alpha-p64                 secondary       Richard Levitte <levitte@openssl.org>
vms-ia64                      secondary       Richard Levitte <levitte@openssl.org>
vms-ia64-p32                  secondary       Richard Levitte <levitte@openssl.org>
vms-ia64-p64                  secondary       Richard Levitte <levitte@openssl.org>
vos-gcc                       unknown
vxworks-mips                  unknown
vxworks-ppc405                unknown
vxworks-ppc60x                unknown
vxworks-ppc750                unknown
vxworks-ppc750-debug          unknown
vxworks-ppc860                unknown
vxworks-ppcgen                unknown
vxworks-simlinux              unknown
```

**Comment by richsalz** [comment]
Please edit the config; I  don't want my name showing up in any print-out.

**Comment by levitte** [comment]
What you're asking for can be understood in several manners:

1.  you wish to remove yourself as supporter of `darwin64-x86_64-cc`.
2.  you wish that `./Configure SUPPORT` wouldn't display your name, or any name.
3.  you wish that `./Configure SUPPORT` wouldn't exist.
4.  you wish that supporters of any config target wasn't implemented as a config attribute, but rather as a comment.

Which is it?

File: Configurations/10-main.conf [hunk 3]
```diff
@@ -521,6 +584,10 @@ my %targets = (
     # HP/UX IA-64 targets
     "hpux-ia64-cc" => {
         inherit_from     => [ "BASE_unix", asm("ia64_asm") ],
+        policy           => 'secondary',
+        supporters       => [ 'appro@openssl.org',
+                              'Paul Dale <paul.dale@oracle.com>' ],
```
Comment (kaduk) [inline Configurations/10-main.conf]: Do we want to leave Andy's address on these if he only wants to be on the hook for "community"-level support?

File: Configurations/10-main.conf [hunk 4]
```diff
@@ -762,12 +870,16 @@ my %targets = (
...
         inherit_from     => [ "linux-generic32", asm("x86_elf_asm") ],
+        policy           => 'unknown',
+
         cflags           => add(picker(release => "-fomit-frame-pointer")),
         cppflags         => add("-DL_ENDIAN"),
         bn_ops           => "BN_LLONG",
     },
     "linux-aout" => {
         inherit_from     => [ "BASE_unix", asm("x86_asm") ],
+        policy           => 'unknown',
```
Comment (kaduk) [inline Configurations/10-main.conf]: Maybe a.out should be pushed into the 'deprecated' bucket?  It's probably not much additional maintenance burden, but are there any current systems for it?

File: Configurations/10-main.conf [hunk 5]
```diff
@@ -1604,26 +1868,35 @@ my %targets = (
...
     },
     "Cygwin-i586" => {
-	inherit_from     => [ "Cygwin-x86" ]
+	inherit_from     => [ "Cygwin-x86" ],
+        policy           => 'unknown',
     },
     "Cygwin-i686" => {
-	inherit_from     => [ "Cygwin-x86" ]
+	inherit_from     => [ "Cygwin-x86" ],
+        policy           => 'unknown',
```
Comment (kaduk) [inline Configurations/10-main.conf]: Do we want to deprecate these pass-through wrappers over Cygwin?

**Comment by richsalz** [comment]
HA!  

I don't wish to be listed as officially supporting Mac.

File: Configurations/90-team.conf [hunk 6]
```diff
@@ -100,6 +116,8 @@ my %targets = (
     },
     "darwin64-debug-test-64-clang" => {
         inherit_from     => [ 'BASE_unix', "x86_64_asm" ],
+        policy           => 'primary', # speculative
```
Comment (kaduk) [inline Configurations/90-team.conf]: These do seem pretty speculative.
While I don't think it's wrong to leave them as 'primary', I would also see a case for marking them 'deprecated'.

**Review by kaduk — APPROVED** [review]
I am a little uncertain about doing this at all, still, as I worry that it will go stale.  But I don't want to hold it up, so you can use a +1 from me, with any or none of the inline items addressed.  (Well, once we figure out what Rich wants.)

**Comment by mattcaswell** [comment]
Drive by comment: I think putting names in the repo is the wrong place for that information. The supporters of a platform may change over time, and on a different timescale to the lifetime of a release. The best we can say is that there was at least one named supporter at the time a release was created. We can maintain the list of current supporters somewhere else (e.g. website)

**Comment by levitte** [comment]
@kaduk:
> I am a little uncertain about doing this at all

Because our [roadmap](https://www.openssl.org/policies/roadmap.html) says so:

> 1. Classify all the platforms known by Configure according to the  platform policy (Timescale: Next feature release)

("Next feature release" is 1.1.1)

@mattcaswell:

> Drive by comment: I think putting names in the repo is the wrong place for that information. The supporters of a platform may change over time, and on a different timescale to the lifetime of a release. The best we can say is that there was at least one named supporter at the time a release was created. We can maintain the list of current supporters somewhere else (e.g. website)

Did you really mean "in the repo"?  Any change there is only a PR away, at least if you mean "git repo", which is frankly the first thing that comes to mind when someone says "repo" to me.  I suspect you meant "in a tarball"...

However, following your line of thought, I'd say that the classification should be placed in the config files either, as that may change just as dynamically...  except for deprecation, I do think that should be marked quite clearly.

A possibility could be to have a "platform status" page alongside the platform policy, or a section as part of the platform policy page, how does that sounds to you?

**Comment by mattcaswell** [comment]
> Did you really mean "in the repo"? Any change there is only a PR away, at least if you mean "git repo", which is frankly the first thing that comes to mind when someone says "repo" to me. I suspect you meant "in a tarball"...

Well, it depends how you look at it. Going in the repo is a first step to going in a tarball - at which point it becomes frozen.

> A possibility could be to have a "platform status" page alongside the platform policy, or a section as part of the platform policy page, how does that sounds to you?

Yes, sounds good.

**Review by t-j-h — APPROVED** [review]
This is in decent enough shape IMHO to go in ...

**Comment by mattcaswell** [comment]
Looks like this missed the freeze :-(

**Comment by levitte** [comment]
Ah, forgot about this one...  that last discussion about this stuff being better placed as a web page had me change my mind and go for that...  I've a change for a web page lined up, just need to finish it.

**Comment by mspncp** [comment]
> Ah, forgot about this one... that last discussion about this stuff being better placed as a web page had me change my mind and go for that... I've a change for a web page lined up, just need to finish it.

@levitte does this mean this pull request has become obsolete?  Anyway, it has conflicts, so I removed the 'ready' label.

**Comment by levitte** [comment]
I'm killing this, as it's not really the right place after all.  The web is a better place, and I do have an unpublished branch to finish.

**Pull request closed without merging**